src/applications/vaos/components/review/ReviewDirectScheduleInfo.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { getTypeOfCare } from '../../utils/selectors';
+import ReasonForAppointmentSection from './ReasonForAppointmentSection';
+import ContactDetailSection from './ContactDetailSection';
+import AppointmentDate from './AppointmentDate';
+
+export default function ReviewDirectScheduleInfo({ data, facility, clinic }) {
+  return (
+    <div>
+      <h1 className="vads-u-font-size--h2">
+        Review your appointment
+        <br />
+        details
+      </h1>
+      <AppointmentDate
+        dates={data.calendarData.selectedDates}
+        systemId={data.vaSystem}
+      />
+      <hr />
+      <h2 className="vaos-appts__block-label vads-u-margin-top--2">
+        {getTypeOfCare(data)?.name}
+      </h2>
+      <hr />
+      <h3 className="vaos-appts__block-label">
+        {clinic?.clinicFriendlyLocationName || clinic?.clinicName}
+      </h3>
+      {facility?.institution.authoritativeName}
```

Comment:
You should always have a facility (and a clinic), so the `?` after them shouldn't be necessary. If someone gets to this page without those values, something has gone wrong already.

Reply:
Done.